src/component/clock.tsx

```diff
@@ -0,0 +1,70 @@
+import { Component } from 'react';
+
+type State = {
+  today: string;
+};
+
+type Props = {
+  date: number;
+};
+
+function getToday(): string {
+  return (new Date()).toUTCString().slice(-12, -4);
+}
+
+function getRandomName(date: number): string {
+  const value = date.toString().slice(-4);
+
+  return `Clock-${value}`;
+}
+
+export class Clock extends Component<Props, State> {
+  state: Readonly<State> = {
+    today: getToday(),
+  };
+
+  timerIdClockName = 0;
+
+  timerIdToday = 0;
+
+  componentDidMount(): void {
+    this.timerIdToday = window.setInterval(() => {
+      this.setState({ today: getToday() });
+      // eslint-disable-next-line no-console
+      console.info(this.state.today);
+    }, 1000);
```

Comment:
you need use interval for ranaming clockName and set it to `App` state `clockName` like on your example:

`const timerId = window.setInterval(() => {
    clockName = getRandomName();
  }, 3300);`
  
  
![image](https://user-images.githubusercontent.com/70009205/196037314-76a02fd0-3846-45f1-af22-fa00cfb82a47.png)